commands/Tools/artist.js

```diff
@@ -0,0 +1,36 @@
+// Requires getSpotifyToken task
+const { Command } = require('klasa');
+const fetch = require('node-fetch');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			cooldown: 10,
+			description: 'Searches Spotify for an artist, and returns an embed.',
+			usage: '<query:str{1,100}>'
+		});
+	}
+
+	async run(msg, [query]) {
+		if (!this.client._spotifyToken) {
```

Comment:
This shouldn't throw... It should try getting the task and running it, not tell you to `wait a bit`

Reply:
the task runs on startup, and every hour, if this statement has triggered then the request itself is probably not working and a wtf error should be emitted instead, as it should never be undefined